packages/puppeteer-core/src/bidi/Page.ts

```diff
@@ -728,6 +728,12 @@ export class BidiPage extends Page {
   override isDragInterceptionEnabled(): boolean {
     return false;
   }
+
+  override async setCacheEnabled(enabled?: boolean): Promise<void> {
+    await this._client().send('Network.setCacheDisabled', {
```

Comment:
Let's add a todo like in other places (as we are doing CDP directly so it won't work for mparch)